test/unit/components/email-step.test.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {shallowWithIntl} from '../../helpers/intl-helpers.jsx';
+import EmailStep from '../../../src/components/join-flow/email-step.jsx';
+import JoinFlowStep from '../../../src/components/join-flow/join-flow-step.jsx';
```

Comment:
This file uses a mix of `import` and `require`. I think `require` is the broadly used way in this repo, so we should consolidate on that.

Reply:
Good call. done!